docs/mons.md

```diff
@@ -7,3 +9,27 @@ kubectl rook-ceph mons
 
 # 10.98.95.196:6789,10.106.118.240:6789,10.111.18.121:6789
 ```
+
+## Restore Quorum
+
+Mon quorum is critical to the Ceph cluster. If majority of mons are not in quorum,
+the cluster will be down. If the majority of mons are also lost permanently,
+the quorum will need to be restore to a remaining good mon in order to bring
+the Ceph cluster up again.
+
+To restore the quorum in this disaster scenario:
+1. Identify that mon quorum is lost. Some indications include:
+   - The Rook operator log shows timeout errors and continuously fails to reconcile
+   - All commands in the toolbox are unresponsive
+   - Multiple mon pods are likely down
+2. Identify which mon has good state (TBD)
```

Comment:
To identify which mon has good state, a user can exec into the mon pod and check its status using the admin socket. For example:
```
1. Locate admin socket file: # ls /var/run/ceph/
2. Check `mon_status`  using : # ceph daemon /var/run/ceph/<mon_admin_socket_file> mon_status
```

Reply:
Good point, thanks